.github/workflows/release.yml

```diff
@@ -25,14 +26,14 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Created by Actions automatically
       with:
         tag_name: ${{ github.ref }}
-        release_name: Release ${{ github.ref }}
+        release_name: ${{ github.ref }}
         body: |
-          Changes in this Release
-          - Release note
+          Changes in this Release:
+          - Please update items included in this release
         draft: true
```

Comment:
why true? I think we know what we are doing when we add a new tag. Also for prerelease - I would rather default to a public release and we could "hide" it manually then if we see there are issues

Reply:
I thought something body update was necessary, but the body can be blank